T/TulipIndicators/build_tarballs.jl

```diff
@@ -0,0 +1,35 @@
+using BinaryBuilder
+
+name = "TulipIndicators"
+version = v"0.9.1"
+hash = "85147c3db868dca9880f78b26de9f4fcecf95fb9" # last rev as of 2022-07-14
+
+sources = [
+	GitSource("https://github.com/TulipCharts/tulipindicators.git", hash)
+]
+
+deps = [
+	HostBuildDependency("Tcl_jll")
+]
+
+script = raw"""
+if ! command -v tclsh &> /dev/null
+then
+	cd ${host_bindir}
+	ln -s tclsh* tclsh
+fi
+cd $WORKSPACE/srcdir/tulipindicators
+make tiamalgamation.c
+gcc -fPIC -shared -Wall -Wextra -Wshadow -Wconversion -std=c99 -pedantic -O2 -g tiamalgamation.c -o libindicators.so
+mkdir -p ${prefix}/include ${libdir}
+mv -t ${libdir} libindicators.so
```

Comment:
```suggestion
mkdir -p ${includedir} ${libdir}
cc -fPIC -shared -Wall -Wextra -Wshadow -Wconversion -std=c99 -pedantic -O2 -g tiamalgamation.c -o ${libdir}/libindicators.${dlext}
```
but you aren't putting anything in `${includedir}`, are you?

Reply:
Nope includedir not needed, thanks for catching that. I was using includedir in a previous version of the script but no longer need it. That should be removed.

Thanks for the general improvements. I wasn't aware of `cc`, wrt to your other comment if `cc` works on windows and mac (posix compliant?) it should work as tulip doesn't have any dependencies. I tested building existing targets + windows and it works. I think mac would work too but couldn't test locally because I haven't accepted the thing